distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -550,26 +559,143 @@ type Column
                 op_result = self.make_op "IIF" [when_true, when_false] new_name
                 adapt_unified_column op_result common_type
 
-    ## Rounding values is not supported in database columns.
-    round : Integer -> Boolean -> Problem_Behavior -> Column | Illegal_Argument | Invalid_Value_Type
-    round self decimal_places=0 use_bankers=False on_problems=Report_Warning =
-        _ = [decimal_places, use_bankers, on_problems]
-        Error.throw <| Unsupported_Database_Operation.Error "`Column.round` is not implemented yet for the Database backends."
+    ## PRIVATE
+
+       Creates a column from a single constant value. The value must be numeric,
+       text, or boolean.
+
+       This method is not static; it must be called on an existing column. That
+       existing column is only used for its internal state; the contents of the
+       column are not used.
+
+       Arguments:
+       - value: the constant value to use for the entire column. Must be
+         numeric, text, or boolean.
+
+       > Example
+         Create a column of the value 42
+
+         column.const 42
+    const : Any -> Column
+    const self value =
```

Comment:
Like James said, it would be cool to integrate this with the expressions part.

See `make_constant` in `evaluate_expression`.

I think we can unify these - let's merge the code from this function and form `make_constant`, then we can just pass `self.make_constant` in `evaluate_expression`.

Reply:
As an additional task, to make all of this consistent, we could also have an analogous `make_constant` method on the in-memory `Table`.

Still, at least for now I think they should be kept `PRIVATE`. We can mention that the created column has the same length as `self`.

If you want to do it in-memory, for now you can just repeat the value `row_count` times. Ideally, we want to rely on O(1) storage, but that is currently only implemented for the Long value type (`LongConstantStorage`) - we'd need to implement these for all other supported in-memory types - which probably exceeds the scope of this PR - but is a nice thing to do in the not-so-distant future.